### DISCUSSION TITLE
Merging issues

### DISCUSSION THREAD

**Discussion by fionaperr318** [Vian1986/Tafe22S108#7] [General]
Hi @Vian1986 ,do you have any idea of how to merging my branch with your development branch?
I don't have local master or develop branch, just wondering how can I merge remotely? I can not use "Clone" and open your codes with visual studio, I 've tried copy the codes about the menu part but then there are plenty errors and conflicts.

**Comment by Vian1986** [comment]
You can merge your branch by below steps:
![image](https://user-images.githubusercontent.com/95561112/160261365-ccb06b03-3b04-4f39-a0b9-3485f3ae2ccb.png)
1. Open tortoisegit to see the log message. **choose all branches** and right click on the working tree changes and choose "fetch". Doing this you will be able to see local master and develop branch
2.  Right click on 5dfef76f7d0b3b499ec756e65a136f96ccbd4e1a and choose **switch/Check out to this** to create your local branch. 
3. Open solution (UniversalCalculator/Calculator.sln) in visual studio, add a new page to the calculator project, name it currency converter and put your code inside.
4. Commit, sync and merge.
Hope this helps. Let me know if you have any questions :)

**Comment by fionaperr318** [comment]
[](url
![110](https://user-images.githubusercontent.com/102396176/160262538-9c952aa9-ae12-4c61-979a-7f78ebaab1fd.PNG)
![113](https://user-images.githubusercontent.com/102396176/160262540-5ee7f30f-cebf-44d3-bf4d-f2dd2e459515.PNG)
)

Thanks for your reply, I realized the problems seem I can not merge to the develop branch as my branch currency conversion is not from the same master branch, I will troubleshoot this now.

**Reply by Vian1986** [reply]
Yep, you just need to develop your currency convertor branch from the revision I mentioned above (5dfef76f7d0b3b499ec756e65a136f96ccbd4e1a ) and then merge
